tests/acceptance/sprint_settings.feature

```diff
@@ -11,3 +33,26 @@ Feature: Sprint Settings
     And I press "save-sprint-settings"
     Then I should see "The sprint settings have been updated"
     And the "ignore_estimates" checkbox should be checked
+
+  Scenario: Invalid data
+    Given I am logged in
+    And a sprint "Sprint 42" exists for the "Wikidata" project
+    And the sprint "Sprint 42" starts on "2014-11-25" and ends on "2014-12-08"
+    When I go to the "Sprint 42" sprint overview
+    And I fill in "sprint_start" with "26.11.2014"
+    And I fill in "sprint_end" with "2014-11-20"
+    And I fill in "title" with ""
+    And I press "save-sprint-settings"
+    Then I should see "The sprint start does not match the format Y-m-d."
+    And I should see "The sprint end must be a date after 2014-11-25."
+    And I should see "The title field is required."
+
+  Scenario: Duplicate title
+    Given I am logged in
+    And a sprint "Sprint 42" exists for the "Wikidata" project
+    And a sprint "Sprint Foo Bar" exists for the "Wikidata" project
```

Comment:
Something weird happens here. Since you previously renamed "Sprint 42" to "Sprint Foo Bar", creating a new "Sprint 42" here causes both sprints to point to the same Phabricator project. It would be good to revert the renaming before doing something like this.